pkgs/data/fonts/victor-mono/default.nix

```diff
@@ -0,0 +1,26 @@
+{ lib, fetchFromGitHub }:
+
+let
+  _name = "victor-mono";
```

Comment:
I would personally prefer `pname` oven `_name` though if other font packages do this then it would be fine to leave it.

Reply:
Don't mind either way, noticed other font packages using pname, will update